src/code.ts

```diff
@@ -42,6 +42,18 @@ export class Code {
   static fromExtendedJSON(doc: CodeExtended): Code {
     return new Code(doc.$code, doc.$scope);
   }
+
+  /** @internal */
+  [Symbol.for('nodejs.util.inspect.custom')](): string {
+    return this.inspect();
+  }
+
+  inspect(): string {
+    const codeJson = this.toJSON();
+    return `{ "code" : "${codeJson.code}"${
```

Comment:
This one, as well as `MaxKey`, `MinKey` seem to output some form of extended JSON. I'm sure this is a "shell-ism", but I think we should be consistent in our representation using `ClassName(...)`. Perhaps you can override this in mongosh to provide legacy shell compat?

Reply:
Sounds like a good idea - changed that for some types already but obviously missed some!